mmv1/relative/file_ops.go

```diff
@@ -0,0 +1,22 @@
+package relative
+
+import (
+	"os"
+	"path/filepath"
+)
+
+// For file IO relative to some base directory
+// Templates and other files are accessed
+// relative to os CWD, this library allows accessing a
+// location relative to an initialized global base
+
+var baseDirectory string
+
+func ReadFile(path string) ([]byte, error) {
```

Comment:
I'm not convinced we need a wrapper function for this, since it's pretty simple behavior; wrapping it may obscure what we're doing. That said, if we do need a wrapper, we should probably do it as a helper on the loader rather than a new util function (unless we have specific plans to reuse this.)